generators/client-2/templates/_webpack-dev

```diff
@@ -0,0 +1 @@
+npm run webpack-dev
```

Comment:
Actually that's what I do all the time but if you don't know the command you have a script to help you.